### ISSUE TITLE
D435i On Turtlebot3

### ISSUE THREAD

**Issue by MART98** [issaiass/realsense2_description#1]
@issaiass Hi ... 
thanks for the information.
I'm doing D435 with no problems. 
But now I want to use the D435i and I don't know how? There are not all of the D435i files in your repository . 
 After that I wanted to simulate D435i on turtlebot3 in gazebo. I need your help . I cannot continue my project because of these problems. i can't find anything about it on the internet

**Comment by issaiass** [comment]
So next you could see my repo of jetbot_perception_2d to use as an object detector.

If you want to use it in 3D you could see the PCL... I will write a few examples probably the next month or this one.

PCL is well documented, you will not get any problems.

**Comment by MART98** [comment]
@issaiass Thanks...
According to your explanation on youtube, how can I activate pointcloud D435?

**Comment by ulatinaedupa** [comment]
No, its is active, if you want to see it, select PCL2 and the corresponding publisher on the box field.

If you want to code using PCL library https://pcl.readthedocs.io/projects/tutorials/en/latest/reading_pcd.html#reading-pcd this example reads a fixed pcl and write to a file.... you can start from there

**Comment by issaiass** [comment]
No, its is active, if you want to see it, select PCL2 and the corresponding publisher on the box field.

If you want to code using PCL library https://pcl.readthedocs.io/projects/tutorials/en/latest/reading_pcd.html#reading-pcd this example reads a fixed pcl and write to a file.... you can start from there

**Comment by MART98** [comment]
@issaiass Really?? Point cloud D435 is activate? But i cant see that...! 
Please help me ... Time is very important for me!!!

**Comment by issaiass** [comment]
@MART98 open rviz, select PointCloud2 and finally select the topic 

That's all

**Comment by MART98** [comment]
![photo6048654060330006212](https://user-images.githubusercontent.com/75748523/115271339-7955fa00-a152-11eb-881f-325d198858e1.jpg)

**Comment by MART98** [comment]
![photo6048654060330006212](https://user-images.githubusercontent.com/75748523/115271410-9094e780-a152-11eb-8d5c-5aa55970c14b.jpg)


The right part of the image is colorful, is Poincloud?
I want the pointclouds to be like this : 

![unnamed](https://user-images.githubusercontent.com/75748523/115271990-26c90d80-a153-11eb-99c9-9e75eb97a79a.png)


What should I do for That?

**Comment by MART98** [comment]
@issaiass thanks
 How can i add this camera D435 to turtlebot3 burger?

**Comment by issaiass** [comment]
You have to add the camera component using urdf or xacro... have to look at the frame that you want to add it
